lib/react-server-route.jsx

```diff
@@ -16,14 +14,14 @@ module.exports = function(req, res, next) {
   // This is essentially a callback lookup. If the requested URL is a known
   // URL based on the routing map as defined in routes.jsx, then this will
   // lead to render properties that can be used to generate page components.
-  ReactRouter.match({ routes, location }, (error, redirectLocation, renderProps) => {
```

Comment:
No, what I mean is as a lowercase method with no parent, it feels like it's against some sort of convention to be here. "match" is such a general term, I can't instinctively look at it and know what it's referring to. Pedantic, I know, and probably a conversation for another place.